PowerEditor/src/MISC/md5/md5Dlgs.cpp

```diff
@@ -255,6 +262,27 @@ INT_PTR CALLBACK MD5FromTextDlg::run_dlgProc(UINT message, WPARAM wParam, LPARAM
 	return FALSE;	
 }
 
+LRESULT MD5FromTextDlg::MD5TextEditRunProc(HWND hwnd, UINT message, WPARAM wParam, LPARAM lParam)
+{
+	switch (message)
+	{
+		case WM_CHAR:
+		{
+			// Select All if Ctrl + A
+			if (wParam == 1)
+			{
```

Comment:
It seems wired to me that wParam value being 1 means `Ctrl + A`. Any online document of MSDN about that?